index.html

```diff
@@ -148,6 +148,35 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Angel Le</p>
+          <p class="contact">
+            <i class="fa-brands fa-facebook"></i>
+            <a href="https://www.facebook.com" target="_blank">https://www.facebook.com/angelxuanle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">https://github.com/lexuanangel</a>
```

Comment:
😅 You'll want to fix this link the same way you need to fix your Facebook link that I commented on above.

Reply:
I have made this change.